src/django/api/reports/facilities_marked_closed.sql

```diff
@@ -0,0 +1,4 @@
+SELECT to_char(approved_at, 'YYYY-MM') as month, COUNT(*) as facilities_closed_in_month
+FROM api_HistoricalFacilityActivityReport
+WHERE status = 'CONFIRMED'
```

Comment:
I think we also want to have `AND closure_state = 'CLOSED'` here so that we do not count a confirmed reopening report.

Reply:
Updated in 44a9a1f